app/src/main/java/com/duckduckgo/app/email/ui/EmailProtectionSignInFragment.kt

```diff
@@ -158,7 +158,7 @@ class EmailProtectionSignInFragment : EmailProtectionFragment(R.layout.fragment_
     }
 
     private fun setClickableSpan(
-        view: MaterialTextView,
+        view: TextView,
```

Comment:
Changed to TextView because we are now using the design system here for consistency across all the waitlists.